### PR TITLE
feat(pings): delete pings subcollection function added

### LINKED ISSUE #1361: Monitor deleted, delete related pings
When deleting a monitor, need to delete all the related pings

### PR DESCRIPTION
**Pull request by webkhushboo**
closes #1361 

### Notes

A summary of what was achieved in this PR

- [ ] Delete pings documents under sub collection once monitor deleted
- [ ] Delete monitor issue fixed

### CODE REVIEW

File: functions/src/monitor/monitor.ts [hunk 1]
```diff
@@ -18,9 +20,10 @@ export const getMonitorPings: any = async (projectUid: string, monitorUid: strin
     .collection('projects')
     .doc(projectUid)
     .get();
-  const project: ProjectModel = <ProjectModel> document.data();
+  const project: ProjectModel = <ProjectModel>document.data();
```
Comment (eddiejaoude) [inline functions/src/monitor/monitor.ts]: When type casting its easier to read with a space

```typescript
<ProjectModel> document.data();
```

File: functions/src/monitor/monitor.ts [hunk 2]
```diff
@@ -41,8 +44,18 @@ export const getMonitorPings: any = async (projectUid: string, monitorUid: strin
...
+    .set(pingResult);
 
   return pingResult;
 }
+
+export const deleteMonitorPings: any = async (projectUid: string, monitorUid: string) => {
+  const documents: DocumentReference[] = await FirebaseAdmin.firestore()
+    .collection(`projects/${projectUid}/${monitorUid}`)
+    .listDocuments()
+  documents.forEach((doc: any) => {
```
Comment (eddiejaoude) [inline functions/src/monitor/monitor.ts]: Should the type here be a `PingModel`

File: functions/src/monitor/monitor.ts (hunk 1 above)
Comment (webkhushboo) [reply functions/src/monitor/monitor.ts]: Done

File: functions/src/monitor/monitor.ts (hunk 2 above)
Comment (webkhushboo) [reply functions/src/monitor/monitor.ts]: PingModel is not application for doc type here. Please provide your inputs on it.

**Comment by eddiejaoude** [comment]
Is this PR out of date, as the next one includes this one and more changes? Should we close this one and just use the next one?

**Comment by webkhushboo** [comment]
Yes this PR is out of date. We can close it and changes will be part of 
https://github.com/DashboardHub/PipelineDashboard/pull/1364

**Pull request closed without merging**